codec/decoder/core/src/decode_slice.cpp

```diff
@@ -52,6 +52,11 @@
 
 namespace WelsDec {
 
+#if defined(_DEBUG)
+FILE* pFile = fopen ("Y:/p4/CodeProjects/Upwork/screencast/xiaotiansf/openh264-1/MV_CAVLC.txt", "w");
```

Comment:
There's no point in first adding this in one commit and then disabling it in a later commit.

Reply:
@thaytan Yes, I'll appreciate it if you can help me  with git operations to generate 2 clean branches for PRs.

Reply:
@xiaotianshi2  you can consider doing squash to merge commits. https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

Reply:
@xiaotianshi2 and please separate commits for different purpose to different PRs, you can do that by starting from a clean branch, then "cherry-pick" the needed commits.